doc/import_production_data.md

```diff
@@ -0,0 +1,23 @@
+# Import Production Data
+
+The content_performance_manager database is no longer imported by default.
+
+To import production data, first either download the data as documented in 
+[the GOV.UK developer docs](https://docs.publishing.service.gov.uk/manual/get-started.html#7-import-production-data)
+
+OR
+
+Get someone to give you a database dump if you don't have access.
+
+Then from inside the development VM run
```

Comment:
This seems to be repeating the tech docs guide - which will be confusing if process changes.